src/clj/rems/middleware.clj

```diff
@@ -22,7 +22,8 @@
             [taoensso.tempura :as tempura])
   (:import (javax.servlet ServletContext)))
 
-(def +default-language+ :en)
+(def +default-language+ (or (:default-language env)
```

Comment:
Can also write this `(:default-language env :en)`

Reply:
Nice suggestion, didn't know that was a thing.